action.yml

```diff
@@ -68,7 +68,12 @@ runs:
       shell: bash
       run: |
         latest_commit_message=$(git log -1 --pretty=format:%s)
-        if [[ $latest_commit_message == "chore: release v"* ]]; then
+        if [[ ${{ inputs.prerelease }} ]]; then
+          pattern="chore: release v[0-9]\+\.[0-9]\+\.[0-9]\+-[0-9]\+$"
```

Comment:
do the + signs require escaping in bash?

Reply:
This is actually specific to grep. And yes, I do believe that they need escaping.

Reply:
And then the periods need the `\` to not signify any character. Is that removed in grep? Does it need to be double escaped?